### PR TITLE
pomsky: Add version 0.11

### PR DESCRIPTION
**Pull request by chillcicada**
pomsky is a portable, modern regular expression language, see more info at [here](https://pomsky-lang.org).

And pomsky is dual-licensed under the [MIT license](https://opensource.org/licenses/MIT) or the [Apache 2.0 license](https://opensource.org/licenses/Apache-2.0).

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/pomsky.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "version": "0.11",
+    "description": "Pomsky is a new, portable, regular expression language",
+    "homepage": "https://pomsky-lang.org",
+    "license": "Dual-licensed under the MIT license or the Apache 2.0 license.",
```
Comment (chawyehsu) [inline bucket/pomsky.json:5]:
```suggestion
    "license": "MIT|Apache-2.0",
```

**Comment by chillcicada** [comment]
should i close this and create a new pr to prune the commit history

**Comment by chawyehsu** [comment]
Up to you, commits will be squashed anyways.

**Pull request closed without merging**